src/project/ProjectManager.js

```diff
@@ -635,7 +635,7 @@ define(function (require, exports, module) {
         if (!projectRoot) {
             return;
         }
-        model.setScrollerInfo($projectTreeContainer[0].scrollWidth, $projectTreeContainer.scrollTop(), $projectTreeContainer.scrollLeft(), $projectTreeContainer.offset().top);
+        model.setScrollerInfo($projectTreeContainer[0].scrollWidth, $projectTreeContainer.scrollTop(), 0, $projectTreeContainer.offset().top);
```

Comment:
Can you please add a comment above this line:

```js
// XXXbramble: We don't want to allow the user to scroll off the right, hiding the files
```

This will help us remember that we changed it in response to a bug